README.md

```diff
@@ -33,17 +33,18 @@ This Executor offers a GPU tag to speed up encoding. For more information on how
 
 ## How to finetune pretrained-model?
 
-### install finetuner
+### Finetune pretrained-model with finetuner
+#### install finetuner
 
 ```bash
 $ pip install finetuner
 ```
 
-### prepare dataset
+#### prepare dataset
```

Comment:
is this the same part as the `prepare dataset` in finetune part? If so, you can remove it.